specification/network/resource-manager/Microsoft.Network/stable/2024-01-01/examples/NetworkWatcherFlowLogCreate.json

```diff
@@ -10,6 +10,7 @@
       "properties": {
         "targetResourceId": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/networkSecurityGroups/desmondcentral-nsg",
         "storageId": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Storage/storageAccounts/nwtest1mgvbfmqsigdxe",
+        "enabledFilteringCriteria": "srcIP=158.255.7.8 || dstPort=56891",
```

Comment:
Why is this needed in a PUT request? PUT should just create a flowLog resource. 

enabledFilteringCriteria is said to be for filtering flowlogs

Same in response too.

Reply:
We expect this condition to be passed during flowlogs creation step. we will collect logs according to this property set on the flowlogs resource.